roles/content_view_publish/tasks/main.yml

```diff
@@ -0,0 +1,21 @@
+---
+# only publish new version
+- name: "Publish new version of Content-View"
+  theforeman.foreman.content_view_version:
+    username: "{{ foreman_user }}"
+    password: "{{ foreman_password }}"
+    server_url: "{{ foreman_server_url }}"
+    organization: "{{ foreman_organization }}"
+    content_view: "{{ foreman_content_view }}"
```

Comment:
In the other roles, we usually followed the rule that the role takes a list to work on, as otherwise there is little benefit over the normal module.